ha/core/support_bundle/ha_bundle.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import os
+
+from cortx.utils.schema.payload import Tar
+from cortx.utils.process import SimpleProcess
+from cortx.utils.log import Log
+
+from ha import const
+from ha.core.error import SupportBundleError
+
+class HABundle:
+
+    def __init__(self):
+        """
+        This Class generates the Support Bundle for Component HA.
+        This method will generate bundle for CSM and include the logs in it.
+        :param command: Csm_cli Command Object :type: command
+        :return:
+        """
+        # TODO Add setup detection logic
+        self._setup_type = "ldr"
```

Comment:
Remove this. This implementation is going to Cortx-1.0 only. For main branch, different implementation will be required.

Reply:
No need to merge this in main branch then.

Reply:
@ajaysrivas 

currently we not having any detection logic 
if we want to merge in main branch

ha_cmd output will not work 
pcs status 

How can we handle this in main branch ?
with same class

Reply:
There is no pacemaker in main. This code will go to Cortx-1.0 only, I have already mentioned it.

Reply:
oh ok

Removed code.

@indrajitzagade @ajaysrivas 
```
No need to merge this in main branch then.
```
we may need to merge this patch in main branch as we pushing all cortx1.0 fix to main also

though we will not call this code as 
This calls are pass through PcsClusterManager which is dead code in main